cmd/wallet-web/src/router/routes.js

```diff
@@ -184,6 +184,11 @@ export default [
         name: 'block-no-auth',
         component: load('pages/BlockNoAuthPage'),
       },
+      {
+        path: 'initiateIssuance',
```

Comment:
Can we rename this to `initiateProtocol` ? This will be used for presentation along with issuance. Also, need to update page name as well

Reply:
updated